docs/Installation.md

```diff
@@ -138,11 +138,11 @@ pkg install wget
 
 * Install Zonemaster Web GUI
 ```sh
-wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip -O temp.zip
+wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip -O zonemaster_web_gui.zip
```

Comment:
@mattias-p, I liked your updates, but when I had to propose other changes in #54 to make installation under FreeBSD I could not resist including your changes with an additional simplification of using `fetch` instead of `wget`. Please see #54 and consider removing the changes to the FreeBSD section from this PR.